### ISSUE TITLE
Poor image quality when running the sample inference command 

### ISSUE THREAD

**Issue by asrlhhh** [autonomousvision/stylegan_xl#68]
Hi Axel, I just set up the environment as instructed and kicked off the inference with the sample command in the readme file:

python gen_images.py --outdir=out --trunc=0.7 --seeds=10-15 --batch-sz 1 \
  --network=https://s3.eu-central-1.amazonaws.com/avg-projects/stylegan_xl/models/imagenet1024.pkl

However the output images are very distorted, especially the ones containing human as foreground. I have uploaded the images from seed 10 to 15 [here](https://drive.google.com/drive/folders/1OL-z4GbXToB3XYTDecAZuSO4cdoGdQYo?usp=sharing). Do you think those results are expected?

**Comment by woctezuma** [comment]
See difficulties met during training by another user: 
- https://github.com/autonomousvision/projected_gan/issues/25

Moreover, the [pre-trained model][pretrained] which you chose was trained on ImageNet.
If you are interested in human faces, try the model trained on FFHQ:
- ```https://s3.eu-central-1.amazonaws.com/avg-projects/stylegan_xl/models/ffhq1024.pkl```

[pretrained]: <https://github.com/autonomousvision/stylegan_xl#pretrained-models>

**Comment by xl-sr** [comment]
see the answer above :)
